fuel-tx/src/transaction/consensus_parameters.rs

```diff
@@ -37,12 +41,20 @@ impl Default for ConsensusParameters {
 }
 
 impl ConsensusParameters {
+    const DEFAULT_BLOCK_TRANSACTION_SIZE_LIMIT: u64 = 126 * 1024;
+
     #[cfg(feature = "test-helpers")]
     /// Constructor for the `ConsensusParameters` with Standard values.
     pub fn standard() -> Self {
         ConsensusParametersV1::standard().into()
     }
 
+    #[cfg(feature = "test-helpers")]
+    /// Constructor for the `ConsensusParameters` with Standard values.
+    pub fn standard_v2() -> Self {
+        ConsensusParametersV2::standard().into()
+    }
```

Comment:
We can just use only `standard`=)

Reply:
Updated in https://github.com/FuelLabs/fuel-vm/pull/821/commits/a1ddd0771f2c9d8994e65098d37f0f197a92ac97